### PR TITLE
Story/gat 1377

### PR DESCRIPTION
**Pull request by GK230**

### CODE REVIEW

**Comment by kymmeh** [comment]
@GK230 is this branch working for you? I'm seeing multiple errors. I've put a PR in to resolve some of them: https://github.com/HDRUK/gateway-web/pull/1273. It's probably best to wait for Peter to review this too as there are conflicts with changes he made.

**Comment by kymmeh** [comment]
Any empty file has been added: `src/pages/search/temp`

**Comment by kymmeh** [comment]
Geeta have you had the design review with Marlon? I think there are some visual tweaks too.

File: src/pages/commonComponents/savedPreferencesModalOld/SavedPreferencesModalOld.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
+import axios from 'axios';
```
Comment (ThinkingMonkeyLtd) [inline src/pages/commonComponents/savedPreferencesModalOld/SavedPreferencesModalOld.js:1]: Is this used anywhere now it has been renamed as old or is it a backup? If it's a backup please get rid of it, these things tend to have a way of being stale and hanging around. If we need it back we can get it from git commits. The same applies to styles.

File: src/pages/search/SearchPage.js [hunk 2]
```diff
@@ -1907,57 +1921,42 @@ class SearchPage extends React.Component {
...
+                                            className='arrow'
+                                            aria-haspopup='true'
                                             onClick={
                                                 this.state.userState[0].loggedIn === false
                                                     ? () => this.showLoginModal()
-                                                    : () => this.setState({ showSavedPreferencesModal: true })
+                                                    : () => this.showSavedPreferencesModal()
                                             }>
-                                            Saved preferences
+                                            Save
```
Comment (ThinkingMonkeyLtd) [inline src/pages/search/SearchPage.js:1933]: This exists in translations

**Review by ThinkingMonkeyLtd — CHANGES_REQUESTED** [review]

**Comment by ThinkingMonkeyLtd** [comment]
Echoing Kymme's comments, branch does not work. Search does not render - if this is a result of doing a merge in github itself when updating an out of date branch, make sure you pull it down to check it works.

File: src/pages/search/SearchPage.js [hunk 3]
```diff
@@ -40,7 +38,12 @@ import FilterSelection from './components/FilterSelection';
 import PapersSearchSort from './components/PapersSearchResults/PapersSearchSort';
 import PeopleSearchSort from './components/PeopleSearchResult/PeopleSearchSort';
 import SearchUtilityBanner from './components/SearchUtilityBanner';
-import ToolsSearchSort from './components/ToolsSearchResults/ToolsSearchSort';
+import ToolsSearchSort from './components/ToolsSearchResults/ToolsSearchSort'import searchService from '../../services/search/search';
```
Comment (ThinkingMonkeyLtd) [inline src/pages/search/SearchPage.js:41]: * ERROR HERE *

**Review by ThinkingMonkeyLtd — CHANGES_REQUESTED** [review]

**Review by kymmeh — DISMISSED** [review]

**Review by dannitahdruk — DISMISSED** [review]

**Review by kymmeh — APPROVED** [review]

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Pull request merged**